Dockerfile

```diff
@@ -4,27 +4,29 @@ LABEL maintainer="Samuel Štancl"
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update \
-    && apt-get install -y curl zip unzip git sqlite3 \
-    php7.2-fpm php7.2-cli \
-    php7.2-pgsql php7.2-sqlite3 php7.2-gd \
-    php7.2-curl php7.2-memcached \
-    php7.2-imap php7.2-mysql php7.2-mbstring \
-    php7.2-xml php7.2-zip php7.2-bcmath php7.2-soap \
-    php7.2-intl php7.2-readline php7.2-xdebug \
+RUN apt-get update
+
+RUN apt-get install -y software-properties-common
```

Comment:
Not sure but maybe we can combine these with the `&&` operator?

Reply:
Good point

Reply:
This is related to the line below too though. And I don't want to combine all 3, so I'll keep this the way it is. I cleaned up a different part of the file.